src/commands/dev/lib/dev-server.ts

```diff
@@ -548,6 +548,7 @@ export default class DevServer {
         const body = await rawBody(req);
         const payload: InvokePayload = {
           method: req.method || 'GET',
+          host: req.headers.host,
```

Comment:
I think this is already available in `getNowProxyHeaders()` below.

Reply:
Same difference really, it's just a property lookup in either case.